packages/logs/README.md

```diff
@@ -8,27 +8,48 @@ Datadog browser logs library.
 
 ### NPM
 
-```
+```javascript
 import { datadogLogs } from '@datadog/browser-logs'
 datadogLogs.init({
   clientToken: 'XXX',
   site: 'datadoghq.com',
   forwardErrorsToLogs: true,
-  sampleRate: 100
+  sampleRate: 100,
 })
 ```
 
-### Bundle
+### CDN async
 
-```
-<script src = 'https://www.datadoghq-browser-agent.com/datadog-logs.js'>
+<!-- prettier-ignore -->
+```html
 <script>
-  window.DD_LOGS.init({
-    clientToken: 'XXX',
-    site: 'datadoghq.com',
-    forwardErrorsToLogs: true,
-    sampleRate: 100
-  });
+ (function(h,o,u,n,d) {
+   h=h[d]=h[d]||{q:[],onReady:function(c){h.q.push(c)}}
+   d=o.createElement(u);d.async=1;d.src=n
+   n=o.getElementsByTagName(u)[0];n.parentNode.insertBefore(d,n)
+})(window,document,'script','https://www.datadoghq-browser-agent.com/datadog-logs.js','DD_LOGS')
+ DD_LOGS.onReady(function() {
+    DD_LOGS.init({
+      clientToken: 'XXX',
+      site: 'datadoghq.com',
+      forwardErrorsToLogs: true,
+      sampleRate: 100,
+    })
+  })
+</script>
+```
+
+### CDN sync
+
+```html
+<script src="https://www.datadoghq-browser-agent.com/datadog-logs.js">
+  <script>
+    window.DD_LOGS.init({
```

Comment:
While we are at it, we could use the same pattern as RUM:
```suggestion
  window.DD_LOGS &&
    window.DD_LOGS.init({
```
```md
- The `window.DD_LOGS` check is used to prevent issues if a loading failure occurs with the Logs SDK.
```